db/import_column_family_job.cc

```diff
@@ -156,7 +156,7 @@ Status ImportColumnFamilyJob::Run() {
                   file_metadata.largest_seqno, false, file_metadata.temperature,
                   kInvalidBlobFileNumber, oldest_ancester_time, current_time,
                   kUnknownFileChecksum, kUnknownFileChecksumFuncName,
-                  kDisableUserTimestamp, kDisableUserTimestamp);
+                  kDisableUserTimestamp, kDisableUserTimestamp, "");
```

Comment:
Is unique_id not available for imported files?

Reply:
I think it's hard to get the unique id during column family import, but let me check again.